javaslang-benchmark/src/test/java/javaslang/collection/VectorBenchmark.java

```diff
@@ -34,30 +39,34 @@ public void testAsserts() {
     }
 
     public static void main(String... args) {
+        JmhRunner.runDebugWithAsserts(CLASSES);
         JmhRunner.runNormalNoAsserts(CLASSES);
     }
 
     @State(Scope.Benchmark)
     public static class Base {
-        @Param({ "32", "1024", "32768"/*, "1048576"*/ }) // i.e. depth 1,2,3(,4) for a branching factor of 32
+        @Param({"32", "1024", "32768" /*, "1048576", "33554432", "1073741824" */}) /* i.e. depth 1,2,3(,4,5,6) for a branching factor of 32 */
```

Comment:
Benchmark profile removed. To run performance benchmarks you should run static `main()` method in specific class. Memory consumption benchmarks starts with `maven test` automatically.
Also see #1413